src/main/java/grondag/canvas/mixin/MixinModelBlockRenderer.java

```diff
@@ -21,31 +21,28 @@
 package grondag.canvas.mixin;
 
 import org.spongepowered.asm.mixin.Mixin;
-import org.spongepowered.asm.mixin.Overwrite;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfo;
 
 import com.mojang.blaze3d.vertex.PoseStack;
 import com.mojang.blaze3d.vertex.VertexConsumer;
 
+import grondag.canvas.apiimpl.rendercontext.CanvasBlockRenderContext;
 import net.minecraft.client.renderer.block.ModelBlockRenderer;
 import net.minecraft.client.resources.model.BakedModel;
 import net.minecraft.core.BlockPos;
 import net.minecraft.util.RandomSource;
 import net.minecraft.world.level.BlockAndTintGetter;
 import net.minecraft.world.level.block.state.BlockState;
 
-import grondag.canvas.apiimpl.rendercontext.CanvasBlockRenderContext;
```

Comment:
Import shouldn't be moved to respect style rules

Reply:
apologies, my editor was being a bit funny and moved it for me. Should be fixed now.